<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration - GLACCIOS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_pump">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Syringe Pump</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_volume">
      <item row="1" column="0">
       <widget class="QLabel" name="pump_label_volume_withdraw">
        <property name="text">
         <string>Withdrawn (µL):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="pump_label_volume_infuse">
        <property name="text">
         <string>Infused (µL):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="pump_lineEdit_volume_withdraw">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="pump_lineEdit_volume_infuse">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pump_label_rate_infuse">
        <property name="text">
         <string>I Rate(µL/min):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="pump_lineEdit_rate_infuse">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="pump_label_rate_withdraw">
        <property name="text">
         <string>W Rate (µL/min):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="pump_lineEdit_rate_withdraw">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_valve2x3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Double 3-ways Valve</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="valve2x3_label_position">
        <property name="text">
         <string>Position: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="valve2x3_lineEdit_position">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_flowA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Flow Controller A</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>64</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_flowA">
      <item row="0" column="0">
       <widget class="QLabel" name="flowA_label_flow">
        <property name="text">
         <string>Flow (mln/min):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="flowA_lineEdit_flow">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="flowA_lineEdit_control_function">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="flowA_label_control_function">
        <property name="text">
         <string>Ctrl function: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_flowB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Flow Controller B</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_flowB">
      <item row="0" column="0">
       <widget class="QLabel" name="flowB_label_flow">
        <property name="text">
         <string>Flow (mln/min):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="flowB_lineEdit_flow">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="flowB_label_control_function">
        <property name="text">
         <string>Ctrl function: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="flowB_lineEdit_control_function">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_pressure">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="title">
     <string>Pressure Controller</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="pressure_label_pressure">
        <property name="text">
         <string>Pressure (mbar):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pressure_lineEdit_pressure">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_electrovalve1">
       <property name="title">
        <string>Electrovalve 1 VERT</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="ev1_label_position">
           <property name="text">
            <string>Position: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ev1_lineEdit_position">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_electrovalve2">
       <property name="title">
        <string>Electrovalve 2 JAUNE</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="ev2_label_position">
           <property name="text">
            <string>Position: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ev2_lineEdit_position">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>170</y>
      <width>181</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>States</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>252</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_state">
      <item>
       <widget class="QRadioButton" name="state_radioButton_manual">
        <property name="text">
         <string>Manual</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_air">
        <property name="text">
         <string>Outside Air</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_init_calib_step1">
        <property name="text">
         <string>Init calib: Step 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_init_calib_step2">
        <property name="text">
         <string>Init calib: Step 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_isocalib_stdA">
        <property name="text">
         <string>IsoCalib: Std A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_isocalib_stdB">
        <property name="text">
         <string>IsoCalib: Std B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_reset_calib">
        <property name="text">
         <string>Reset calib</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_humdep">
        <property name="text">
         <string>HumDep</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_picarro_code">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Picarro code</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_picarro">
      <item>
       <widget class="QLabel" name="picarro_label_code">
        <property name="text">
         <string>Code:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="picarro_lineEdit_code">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_listen_picarro_code">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Listen picarro code</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="state_radioButton_picarro_listen">
        <property name="text">
         <string>Listen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="state_radioButton_picarro_ignore">
        <property name="text">
         <string>Ignore</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_temperature">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Temperature</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="temperature_label_measure">
        <property name="text">
         <string>Measure (°C): </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="temperature_label_setpoint">
        <property name="text">
         <string>Setpoint (°C): </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="temperature_lineEdit_setpoint">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="temperature_lineEdit_measure">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="tempswitch_label_position">
        <property name="text">
         <string>Switch: </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="tempswitch_lineEdit_position">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_pressure_pump_switch">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Pressure pump</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="ppswitch_label_position">
        <property name="text">
         <string>Switch: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ppswitch_lineEdit_position">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>groupBox_pump</zorder>
   <zorder>groupBox_valve2x3</zorder>
   <zorder>groupBox_flowA</zorder>
   <zorder>groupBox_flowB</zorder>
   <zorder>groupBox_pressure</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_picarro_code</zorder>
   <zorder>groupBox_listen_picarro_code</zorder>
   <zorder>groupBox_temperature</zorder>
   <zorder>groupBox_pressure_pump_switch</zorder>
   <zorder>ev2_label_position</zorder>
   <zorder>flowA_label_control_function</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInstrument">
    <property name="title">
     <string>Expert mode</string>
    </property>
    <addaction name="action_state"/>
    <addaction name="action_maintenancetools"/>
    <addaction name="action_pump"/>
    <addaction name="action_flows"/>
    <addaction name="action_pressure"/>
    <addaction name="action_valve2x3"/>
    <addaction name="action_electrovalves"/>
    <addaction name="action_temperature"/>
    <addaction name="action_debuglog"/>
   </widget>
   <addaction name="menuInstrument"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_pump">
   <property name="text">
    <string>Syringe pump</string>
   </property>
  </action>
  <action name="action_flows">
   <property name="text">
    <string>Flow controllers</string>
   </property>
  </action>
  <action name="action_pressure">
   <property name="text">
    <string>Pressure controller</string>
   </property>
  </action>
  <action name="action_valve2x3">
   <property name="text">
    <string>Double 3-ways valve</string>
   </property>
  </action>
  <action name="action_electrovalves">
   <property name="text">
    <string>Electro-valves</string>
   </property>
  </action>
  <action name="action_maintenancetools">
   <property name="text">
    <string>Maintenance Tools</string>
   </property>
  </action>
  <action name="action_temperature">
   <property name="text">
    <string>Temperature</string>
   </property>
  </action>
  <action name="action_debuglog">
   <property name="text">
    <string>Debug Log</string>
   </property>
  </action>
  <action name="action_state">
   <property name="text">
    <string>State</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pump_lineEdit_volume_infuse</tabstop>
  <tabstop>pump_lineEdit_volume_withdraw</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_valve"/>
 </buttongroups>
</ui>
